core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -3116,5 +3117,48 @@ public static void setLocalDictColumnsToWrapperSchema(List<ColumnSchema> columns
       }
     }
   }
-}
 
+  /**
+   * Utility function to read a whole file as a string,
+   * Must not use this if the file is very huge. As it may result in memory exhaustion.
+   * @param filePath
+   * @return
+   */
+  public static String readFromFile(String filePath) {
```

Comment:
If it is only used for test case purpose, move these to test cases only.

Reply:
ok. done